src/main/webapp/javascript/recipe-rendering.js

```diff
@@ -1,3 +1,19 @@
+// Set listeners for rendering based on user status,
+// except for User Page, which has special listeners.
+if (document.title != "My Recipes") {
+  // Listener that triggers when a new or different user signs in.
+  let userChanged = setIcons;
```

Comment:
Is this supposed to be `setIcons -> setIcons()`?

Reply:
userChanged is the global listener function user in the user-auth file. Here it is set to reference setIcons.